Comparing source compatibility of /home/runner/work/japicmp/japicmp/japicmp/target/japicmp-0.20.0.jar against /home/runner/.m2/repository/com/github/siom79/japicmp/japicmp/0.19.1/japicmp-0.19.1.jar
***! MODIFIED CLASS: PUBLIC japicmp.cmp.JApiCmpArchive  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	---! REMOVED CONSTRUCTOR: PUBLIC(-) JApiCmpArchive(byte[], java.lang.String)
	+++  NEW CONSTRUCTOR: PUBLIC(+) JApiCmpArchive(byte[], java.lang.String, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) java.lang.String getName()
***  MODIFIED ENUM: PUBLIC STATIC FINAL japicmp.exception.JApiCmpException$Reason  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) japicmp.exception.JApiCmpException$Reason ResourceNotFound
+++  NEW CLASS: PUBLIC(+) japicmp.output.html.HtmlOutput  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW CONSTRUCTOR: PUBLIC(+) HtmlOutput(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) java.lang.String getHtml()
+++  NEW CLASS: PUBLIC(+) japicmp.output.html.HtmlOutputGenerator  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW SUPERCLASS: japicmp.output.OutputGenerator
	+++  NEW CONSTRUCTOR: PUBLIC(+) HtmlOutputGenerator(java.util.List<japicmp.model.JApiClass>, japicmp.config.Options, japicmp.output.html.HtmlOutputGeneratorOptions)
	+++  NEW METHOD: PUBLIC(+) japicmp.output.html.HtmlOutput generate()
+++  NEW CLASS: PUBLIC(+) japicmp.output.html.HtmlOutputGeneratorOptions  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW CONSTRUCTOR: PUBLIC(+) HtmlOutputGeneratorOptions()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getSemanticVersioningInformation()
	+++  NEW METHOD: PUBLIC(+) japicmp.util.Optional<java.lang.String> getTitle()
	+++  NEW METHOD: PUBLIC(+) void setSemanticVersioningInformation(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void setTitle(java.lang.String)
+++  NEW CLASS: PUBLIC(+) japicmp.output.html.TemplateEngine  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW CONSTRUCTOR: PUBLIC(+) TemplateEngine()
	+++  NEW METHOD: PUBLIC(+) java.lang.String loadAndFillTemplate(java.lang.String, java.util.Map<java.lang.String,java.lang.String>)
	+++  NEW METHOD: PUBLIC(+) java.lang.String loadTemplate(java.lang.String)
***! MODIFIED CLASS: PUBLIC japicmp.output.xml.XmlOutput  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	---! REMOVED METHOD: PUBLIC(-) japicmp.util.Optional<java.io.ByteArrayOutputStream> getHtmlOutputStream()
	---! REMOVED METHOD: PUBLIC(-) void setHtmlOutputStream(japicmp.util.Optional<java.io.ByteArrayOutputStream>)
+++  NEW CLASS: PUBLIC(+) japicmp.util.StringHelper  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) STATIC(+) java.lang.String filtersAsString(java.util.List<japicmp.filter.Filter>, boolean)

Semantic versioning suggestion: 1.0.0